src/csharp/Microsoft.Spark.E2ETest/IpcTests/ML/Feature/StopWordsRemoverTests.cs

```diff
@@ -0,0 +1,108 @@
+using System.IO;
+using Microsoft.Spark.E2ETest.Utils;
+using Microsoft.Spark.ML.Feature;
+using Microsoft.Spark.Sql;
+using Microsoft.Spark.Sql.Types;
+using Microsoft.Spark.UnitTest.TestUtils;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.IpcTests.ML.Feature
+{
+    [Collection("Spark E2E Tests")]
+    public class StopWordsRemoverTests : FeatureBaseTests<StopWordsRemover>
+    {
+        private readonly SparkSession _spark;
+
+        public StopWordsRemoverTests(SparkFixture fixture) : base(fixture)
+        {
+            _spark = fixture.Spark;
+        }
+
+        [Fact]
+        public void TestStopWordsRemoverWithoutLocale()
+        {
+            var expectedUid = "theUidWithOutLocale";
+            var expectedInputCol = "input_col";
+            var expectedOutputCol = "output_col";
+            var expectedCaseSensitive = false;
```

Comment:
Let's give the explicit type of these variables instead of using `var`

Reply:
@Niharikadutta seen people are suggesting to use var than explicit type, do you have any specific reason for this suggests.?